src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -94,8 +94,12 @@ def preprocess_program(
     return fencil_definition
 
 
-def get_args(params: Sequence[itir.Sym], args: Sequence[Any]) -> dict[str, Any]:
-    return {name.id: convert_arg(arg) for name, arg in zip(params, args)}
+def get_args(params: Union[Sequence[itir.Sym], dace.SDFG], args: Sequence[Any]) -> dict[str, Any]:
```

Comment:
`get_args` and `get_offset_args` are used only in one place. Do we need to keep `Sequence[itir.Sym]` as input type?

Reply:
I was also thinking that, but I kept it in the end for backwards compatibility, but you are right, we should remove it and only accept SDFG.